src/assets/styles/main.scss

```diff
@@ -42,15 +42,6 @@
     @apply text-lg;
   }
 
-  a {
-    &:focus {
-      @apply outline-none;
-      @apply ring-2;
-      @apply ring-primary-100;
-      @apply border-transparent;
-    }
-  }
-
```

Comment:
Want to point this out and see if anyone sees an issue with this. These styles have been tricky to work with for a long time since there are a lot of places we have `a` tags with different styles for focusing.